academic_observatory_workflows/database/sql/create_aggregate.sql.jinja2

```diff
@@ -801,7 +801,7 @@ WITH tmp_disciplines AS
             FROM UNNEST(openalex.concepts) as fields where fields.level = 0)) )) as level0
       ) as disciplines
   FROM
-    `{{ project_id }}.{{ dataset_id }}.doi{{ release_date.strftime('%Y%m%d') }}` as dois,
+    `{{ project_id }}.{{ dataset_id }}.doi{{ snapshot_date.strftime('%Y%m%d') }}` as dois,
```

Comment:
For cases like this in the Jinja template, I wonder if it's better to just supply the fully qualified table ID? It makes the template cleaner and leaves any and all logic to python. What do you think?

Reply:
Update: I see some other templates have been implemented this way ^ Is there a case for not doing the same for this one and its siblings?

Reply:
Yeah it would be better to use the approach that you suggest wherever possible.

Some of those templates only specify a project_id and dataset_id and then there are multiple table names referenced off them, so those instances were a can of worms to sort out.

We could have another crack when we refactor the DOI table?

Reply:
Yeah I think so. Is there a Jira ticket for the DOI refactor? If not I can make one and mention this so it's not forgotten.

Reply:
Thanks that would be great. I can't see one.